app/models/manageiq/providers/google/inventory/parser.rb

```diff
@@ -710,6 +710,8 @@ def parse_port_range(port_range)
   end
 
   def parse_uid_from_url(url)
+    return if url.nil?
```

Comment:
Thought about just adding a `if cloud_volume.zone` to https://github.com/ManageIQ/manageiq-providers-google/blob/2216e3851b690009d82f636b5e089d72b2a45269/app/models/manageiq/providers/google/inventory/parser.rb#L115 but figured we could knock out the whole class of failures if we hit this on something else later on

Reply:
Do callers handle nils properly?  I don't see how this method ever returned nil before, so this would be an interface change, effectively.

Reply:
Sorry should have added that, yes there are 6 callers of this, 4 pass the result to `.lazy_find` which returns `nil` if the arg is `nil`, one sets cloud_volume.volume_type which can be `nil`, and the last does a `==` comparison with a string.